package.json

```diff
@@ -94,7 +94,7 @@
     "tmp-promise": "^3.0.2"
   },
   "engines": {
-    "node": ">=8.3.0"
+    "node": ">=10.3.0"
```

Comment:
`cli` is using `>=10.18.0`

https://github.com/netlify/cli/blob/2a662e4546520f31e003306b2d3e120d021f8ce4/package.json#L14

[Link to builds usage data](https://github.com/netlify/engineering/issues/50#issuecomment-704451333) from 2 months ago. Looking at it, I am wondering whether we should also pick `10.18.0`?

cc @JGAntunes too

Reply:
That works for me. I'll update.